include/boost/serialization/strong_typedef.hpp

```diff
@@ -10,6 +10,7 @@
 // strong_typedef.hpp:
 
 // (C) Copyright 2002 Robert Ramey - http://www.rrsd.com . 
+// (C) Copyright 2016 Ashish Sadanandan
```

Comment:
I'm not sure whether I was supposed to add to the copyright or not. If this isn't the right thing to do, then let me know and I'll change it.